code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -22,6 +22,10 @@
 		apparent_species = ", \an [dna.species.name]"
 	. = list("<span class='info'>*---------*\nThis is <EM>[!obscure_name ? name : "Unknown"][apparent_species]</EM>!")
 
+	if(isobserver(user) || check_rights(R_ADMIN)) // you cannot hide your true name from the ones byond
```

Comment:
make it observer only, admins probably don't really need (or want) this when they're playing, plus they can aghost and have access to the player panel which already will tell you who someone is.

Reply:
as if you would know